README.md

```diff
@@ -1 +1,102 @@
+
+<img src= "PicsForGuide/4 reOpenDC.png" href= "Dev Container drop down menu"/>
+
+### Step 5
+To ensure Dev Container is running properly look back at the bottom left corner of VS code where the icon was, and it should look like the picture below.
+
+<img src = "PicsForGuide/5 IfYouAreinDCbottom.png" href = "Dev Container after Connecting" width = 300px />
+
+### Step 6
+After checking you are in Dev Container and locating the file you want to make changes to you can start looking for the Source Control icon as seen below.
+
+<img src = "PicsForGuide/6 IfYouAreinDCOpenSC.png" href = "Source Control icon" width = 300px />
+
+### Step 7
+On top of ensuring you locate Source Control you also need to make sure you are on the correct branch, or if you dont have a branch how you could make one. To tell if you are on the correct branch you can navagate back down to the bottom left of VS code and look right next to the blue Dev container information. If it says main you are in the wrong branch.
+
+<img src = "PicsForGuide/7 CheckBranch.png" href = "Main branch display"/>
+
+#### Below will be the image guide with some text to making a new branch
+1. Click on where it displays the current branch name in the bottom left. Afterwards another drop down menu will open and you will click on create new branch. (If you are looking to just change branches you can select from existing branches from the same drop down.)
+
+<img src= "PicsForGuide/8 MakeNewBranch.png" href = "Drop down for making new branches"/>
+
+2. Next you would put in a branch name and hit enter as shown below.
+
+<img src = "PicsForGuide/9 EnterBranchName.png" href= "Text box for inputing name"/>
+<img src = "PicsForGuide/10 BranchNameEntered.png" href = "Text box With branch name">
+3. Ensure you are on the correct branch by looking the the bottom left corner again and it should look similar to the image below.
+
+<img src = "PicsForGuide/11 HowItshouldLookonBranch.png" href = "Dev Container with branch name"/>
+4. Finally if you are on the Source Control Tab it should look like the image below. (You can choose to publish the branch now or latter either is fine.)
+
+<img src = "PicsForGuide/12 howSCShouldLook.png" href = "Source Control with new branch" width = 300px />
+
+### Step 8
+Start making changes what ever changes you need to make.
+### Step 9
+After making your changes navigate to the Source Control tab and it will probably look a little like the image below.
+
+<img src = "PicsForGuide/13 afterChanges.png" href = "Source Control with all the changes made since last commt" width = 300px />
+
+Now I know this looks like a lot of stuff. Probably a lot more stuff than what you remember changing. The reason a lot of these files change is due to how the Dev Container works, its constantly modifying the file and changing values for it to work and it is imparative that you only choose the items you want to commit. But first you have to locate them in this list.
```

Comment:
The files being changes have nothing to do with how dev containers work, but rather how the Java extensions work. File can be changed for any number of reasons.